data/json/recipes/recipe_deconstruction.json

```diff
@@ -119,12 +119,7 @@
     "time": 1200,
     "reversible": true,
     "qualities": [ {"id":"HAMMER","level":1} ],
```

Comment:
Missed one. I'd put spaces after the comas too on all the qualities lines. I don't know if anyone ever agreed on that one though so I'd see what @mugling thinks.